packages/core/stories/text/text.qa.stories.tsx

```diff
@@ -91,7 +91,20 @@ export const AllVariantsGrid: StoryFn<QAContainerProps> = (props) => (
 );
 
 AllVariantsGrid.parameters = {
-  chromatic: { disableSnapshot: false },
+  chromatic: {
+    disableSnapshot: false,
+    modes: {
+      theme: {
+        themeNext: "disabled",
+      },
+      themeNext: {
```

Comment:
This didn't work, the QA stories have nested providers that haven't inherited theme next

Reply:
Good that you caught this. the code was not actually working lol